packages/fxa-payments-server/src/routes/Checkout/index.scss

```diff
@@ -73,33 +50,8 @@
     padding: 16px 48px 48px;
   }
 
-  header {
-    display: flex;
-    flex-direction: column;
-    justify-content: center;
-    margin-bottom: initial;
-    margin-top: 20px;
-    padding-bottom: 32px;
-    text-align: center;
-
-    @include min-width('desktop') {
-      margin-top: 32px;
-    }
-
-    p {
-      color: #6d6d6e;
-    }
-  }
-
   .pay-with-heading {
-    color: rgb(115, 115, 115);
-    margin: 0 0 24px 0;
-    display: grid;
-    width: 100%;
-    align-items: center;
-    text-align: center;
     grid-template-columns: minmax(20px, 1fr) auto minmax(20px, 1fr);
-    grid-gap: 20px;
     &:before,
```

Comment:
Investigating - When I converted this to Tailwind the border appeared thicker

Reply:
Keep for now. Borders have to be updated for all components and later subscriptions. Either a new ticket will be filed or added to existing ticket regarding TW global/shared styles.